composer.json

```diff
@@ -1,39 +1,42 @@
 {
-  "name": "dunglas/php-property-info",
-  "type": "library",
-  "description": "Retrieve type and description of PHP properties using various sources ",
-  "keywords": [
-    "property",
-    "type",
-    "PHPDoc",
-    "symfony",
-    "validator",
-    "doctrine"
-  ],
-  "homepage": "http://dunglas.fr",
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "Kévin Dunglas",
-      "email": "dunglas@gmail.com"
+    "name": "dunglas/php-property-info",
+    "type": "library",
+    "description": "Retrieve type and description of PHP properties using various sources ",
+    "keywords": [
+        "property",
+        "type",
+        "PHPDoc",
+        "symfony",
+        "validator",
+        "doctrine"
+    ],
+    "homepage": "http://dunglas.fr",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Kévin Dunglas",
+            "email": "dunglas@gmail.com"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "PropertyInfo\\": "src/PropertyInfo"
+        }
+    },
+    "require": {
+        "php": ">=5.4"
+    },
+    "require-dev": {
+        "phpspec/phpspec": "~2.1",
+        "doctrine/orm": "~2.3",
+        "phpdocumentor/reflection": "~1.0"
```

Comment:
Since this is only a require-dev thing, you can require ^1.0.4 here and leave the conflict, other packages using yours as dependency don't need to know or care about your dev dependencies (and in this case dev conflict in a way).